tests/e2e/precompile-upgrade/suites.go

```diff
@@ -0,0 +1,330 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// Implements precompile upgrade tests for Tx Allow List, requires network-runner cluster.
+package precompile_upgrade
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/plugin/evm"
+	"github.com/ava-labs/subnet-evm/precompile"
+	"github.com/ava-labs/subnet-evm/tests/e2e/utils"
+
+	"github.com/ethereum/go-ethereum/common"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+type chainConfig struct {
+	evm.Config
+}
```

Comment:
what's the purpose of wrapping the `evm.Config` like this? This is a bit confusing because `chainConfig` is also the term used to `*params.ChainConfig`

Reply:
Changing to `evm.Config`